packages/amplify_datastore/example/integration_test/utils/setup_utils.dart

```diff
@@ -34,6 +56,45 @@ Future<void> configureDataStore() async {
 ///
 /// see: https:///github.com/aws-amplify/amplify-android/issues/1464
 Future<void> clearDataStore() async {
-  await Future.delayed(Duration(milliseconds: 100));
+  await Future.delayed(Duration(seconds: 2));
```

Comment:
nit: Does it make sense to make this delay a constant for the whole suite so it can be changed in one place? I believe the non-cloud tests do something similar IIRC.